presto-docs/src/main/sphinx/release/release-0.254.rst

```diff
@@ -0,0 +1,40 @@
+=============
+Release 0.254
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug where queries that have both remote functions and a local function with only constant arguments could throw an IndexOutOfBoundException during planning. The bug was introduced in release 0.253 by :pr:`16039`.
+* Add documentation for Glue Catalog support in Hive. :doc:`/connector/hive`.
+* Add iceberg connector.
```

Comment:
I would maybe add a new section called "Iceberg Changes", similar to other connectors.  and then have the release note "Add new Iceberg Connector", with a link to the documentation.

Reply:
removed this since there's no documentation added